apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -184,6 +183,11 @@ export class UpsertWorkflowUseCase {
     );
   }
 
+  /**
+   * Upsert workflow preferences. While this operation is not typically needed
+   * in a standard workflow update, it's maintained here to support environment
+   * sync scenarios and to enable code reusability.
+   */
```

Comment:
good comment to have :)

Reply:
Removing the comment as prefs are removed from this use-case in https://github.com/novuhq/novu/pull/7005, avoiding a conflict now.